translate/rus.json

```diff
@@ -315,8 +315,8 @@
 "reverses":"обратные",
 "right":"право",
 "rotate board":"повернуть доску",
-"round32":"1/16 финала",
-"round16":"1/8 финала",
+"Round of 32":"1/16 финала",
+"Round of 16":"1/8 финала",
```

Comment:
Found out that some of the TCEC Cup brackets translation is lost, so I had to do some reverts here.

Reply:
Okay, just make sure NOT to modify any English keys, in the "key":"value" pairs.
The English keys cannot be changed because the code is using them, you can change the values of course.

Reply:
So... I am not sure here then, is this applicable in this way or not?